app/controllers/supplejack_api/harvester/sources_controller.rb

```diff
@@ -32,7 +32,7 @@ def show
 
       def update
         @source = Source.find(params[:id])
-        source_params['status_updated_at'] = Time.zone.now.to_datetime if source_params['status']
+        source_params['status_updated_at'] = Time.now.utc.to_datetime if source_params['status']
```

Comment:
This can stay time.zone.now.to_datetime.

Reply:
It doesn't give the same result though :/
```
[4] pry(main)> Time.now.utc.to_datetime
=> Mon, 04 Nov 2019 03:33:19 +0000
[5] pry(main)> Time.zone.now.to_datetime
=> Mon, 04 Nov 2019 16:33:21 +1300
```

To be consistent with the rest of the app, I prefer to keep UTC

Reply:
What do you think?

Reply:
Ok that's all good 👍